Docs for nilu air_quality platform

hfurubotten opened the pull request:
**Description:**

Documentation for new platform in the `air_pollutants` platform. Shows data from the governmental air research institutes sensor stations within Norway. 

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19674

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

hfurubotten committed:
Docs for nilu air_pollution platform

hfurubotten committed:
Updated with new location of pollution index

hfurubotten committed:
Fixed stations to be plural

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

hfurubotten commented:
I'm guessing that I should rename the documentation file from `air_pollutants.nilu.markdown` to `air_quality.nilu.markdown` since the component had the name change in the backend? 

However can't see that the other platforms have gotten the name change after home-assistant/home-assistant#19448 tho. Should I open an issue/PR on this as well? Should probably get the name change before the beta period is done..

hfurubotten committed:
platform rename from air_pollutants to air_quality
Managed to remove a commit from maintainer.
Edits should be within this renaming commit.

fabaff committed:
Update 'ha_release'
